### ISSUE TITLE
Login crash when first loading char list 

### ISSUE THREAD

**Issue by jhlr** [chromiecraft/chromiecraft#4704]
### What client do you play on?

enUS

### Faction

Alliance

### Content Phase:

Generic

### Current Behaviour

I am getting this message whenever I try to login for the first time. I even tried creating another account.

<img width="311" alt="Screen Shot 2022-12-31 at 02 25 08" src="https://user-images.githubusercontent.com/9724763/210126079-d8b058c1-c5df-4505-8d53-1fccab8e64b2.png">

### Expected Blizzlike Behaviour

It was supposed to start the process of creating my first char

### Source

None

### Steps to reproduce the problem

All i did was enter game, login and after it says "loading char list" it crashes

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/c05c45c51d3d414ab3bec40e3e4bc2b5c10a977e

### Operating system

Mac M1 
WS11WineCX64Bit22.0.1

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-bg-item-reward](https://github.com/azerothcore/mod-bg-item-reward)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chat-transmitter](https://github.com/azerothcore/mod-chat-transmitter)
- [mod-chromie-xp](https://github.com/azerothcore/mod-chromie-xp)
- [mod-cta-switch](https://github.com/azerothcore/mod-cta-switch)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna](https://github.com/azerothcore/mod-eluna)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-low-level-rbg](https://github.com/azerothcore/mod-low-level-rbg)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-progression-system](https://github.com/azerothcore/mod-progression-system)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-rdf-expansion](https://github.com/azerothcore/mod-rdf-expansion)
- [mod-transmog](https://github.com/azerothcore/mod-transmog)
- [mod-weekend-xp](https://github.com/azerothcore/mod-weekend-xp)
- [mod-instanced-worldbosses](https://github.com/nyeriah/mod-instanced-worldbosses)
- [mod-zone-difficulty](https://github.com/azerothcore/mod-zone-difficulty)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-exchange-npc](https://github.com/55Honey/Acore_ExchangeNpc)
- [lua-custom-worldboss](https://github.com/55Honey/Acore_CustomWorldboss)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)

### Customizations

None

### Server

ChromieCraft

**Comment by heyitsbench** [comment]
I have a feeling this is due to the M1 part of this.

**Comment by Samsequel** [comment]
Yep, could not reproduce so I fear its due to m1

**Comment by younver** [comment]
hey @Samsequel @heyitsbench,

I've just downloaded chromiecraft following the [instructions](https://www.mangosrumors.org/how-to-run-wow-32-bit-on-mac-os/) as a Macbook Air M2 user, and faced same issue.

Even though I had no addons installed, I've tried cleaning `Interface`, `WTF` and `Cache` folders, but game is still crashing on loading character list.

Also note that I've been playing chromiecraft for a while and had few characters already.

Is there any way to fix this issue?
